config/commands.js

```diff
@@ -1398,7 +1402,7 @@ var commands = exports.commands = {
 		}
 		if (target === 'all' || target === 'tiershift' || target === 'ts') {
 			matched = true;
-			if (target !== 'all') buffer += "Pokémon below OU/BL get all their stats boosted. UU/BL2 get +5, RU/BL3 get +10, and NU or lower get +15.<br />";
+			if (target !== 'all') buffer += "Pokémon below OU / BL get all their stats boosted. UU / BL2 get +5, RU / BL3 get +10, and NU or lower get +15.<br />";
```

Comment:
Yeah, that makes sense. It's mainly a preference thing of mine, so I did it out of habit. :P